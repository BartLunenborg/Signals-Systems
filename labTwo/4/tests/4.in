5: [1,2,3,6,0]
14: [1,4,12,25,35,50,46,54,79,71,43,24,12,0]
