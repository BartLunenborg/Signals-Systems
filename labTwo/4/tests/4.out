10: [1,2,5,3,2,7,8,5,3,2]
